src/simulation/onboardDataHandling/_GeneralModuleFiles/dataStorageUnitBase.h

```diff
@@ -34,7 +34,7 @@
 
 struct dataInstance {
     char dataInstanceName[128];     //!< data instance name
-    double dataInstanceSum;         //!< data instance sum value, bits
+    long long int dataInstanceSum;         //!< data instance sum value, bits
```

Comment:
I realize that this has already been merged, but maybe after the break, this can be changed to use the precise width type `int64_t` or even `uint64_t` if your semantics demand it. We should prefer precise width types for any integer types larger than `int`. The bit size of integral types is not guaranteed across platforms/compilers. It benefits us to er on the side of guarantee.

Reply:
Howdy @LorenzzoQM , @patkenneally  makes some good points here about the use of `int64_t`, etc.  Could you please create a small branch next week to address his feedback?  I'll respond to review request quickly so it doesn't slow down your research tasks.

Reply:
Sure! I will address the comments as soon as possible. Thank you for the feedback, @patkenneally.